dev-tools/buy-numbers.js

```diff
@@ -0,0 +1,57 @@
+const parseArgs = require("minimist");
+import twilio from "../src/server/api/lib/twilio";
```

Comment:
remove dependency on `api/lib/twilio` and just copy the functions over here to avoid needing direct database access to use this CLI once it's rebased. This will make integrating this feature with phone number management much easier.